app/templates/components/component-tree-item.hbs

```diff
@@ -4,17 +4,12 @@
   {{action inspect item.view.objectId}}
 >
   {{#if item.hasChildren}}
-    <button
-      class="component-tree-item__expand {{if item.expanded "expanded" "collapsed"}}"
-      {{action toggleExpanded bubbles=false}}
-    >
-      {{svg-jar
-        "disclosure-triangle"
-        title="Toggle visibility"
-        width="9px"
-        height="9px"
-      }}
-    </button>
+    <DisclosureTriangle
```

Comment:
See https://github.com/emberjs/ember-inspector/pull/925/files#r251897592 !

Reply:
We need to avoid angle bracket components for now, unfortunately.